third_party/repositories/repositories.bzl

```diff
@@ -0,0 +1,38 @@
+load("//third_party/repositories:scala_2_11.bzl", _artifacts_2_11 = "artifacts")
+load("//third_party/repositories:scala_2_12.bzl", _artifacts_2_12 = "artifacts")
+load(
+    "@io_bazel_rules_scala//scala:scala_cross_version.bzl",
+    "default_maven_server_urls",
+    "default_scala_version",
+    "extract_major_version",
+)
+load(
+    "@io_bazel_rules_scala//scala:scala_maven_import_external.bzl",
+    _scala_maven_import_external = "scala_maven_import_external",
+)
+
+artifacts_by_major_scala_version = {
+    "2.11": _artifacts_2_11,
+    "2.12": _artifacts_2_12,
+}
+
+def repositories(
+        scala_version = default_scala_version(),
+        for_artifact_ids = [],
+        maven_servers = default_maven_server_urls(),
```

Comment:
Oh I see you already call here for default value - so can we remove the explicit parameter from caller site?

Reply:
Users will need to be able to provide their own urls